tools/exp/convert/convert.go

```diff
@@ -0,0 +1,113 @@
+package convert
+
+import (
+	"errors"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/aiven/aiven-go-client/tools/exp/types"
+	"github.com/mitchellh/copystructure"
+)
+
+// errUnexpected is the error that is returned when an unexpected error occurs.
+var errUnexpected = errors.New("unexpected conversion error")
+
+// UserConfigSchema converts aiven.UserConfigSchema to UserConfigSchema.
+func UserConfigSchema(v aiven.UserConfigSchema) (*types.UserConfigSchema, error) {
+	var cnp map[string]types.UserConfigSchema = nil
+
+	{
```

Comment:
I do not think it is a good idea to add `{}` scope, as I can see it has nothing regarding the logic. Perhaps the intention was to add readability to the code, though to my eye, it was a distraction.

Reply:
Perhaps adding a comment on each logic block would help, or even better, make it a struct and split logic to smaller methods if the intent is to make it more understandable.